php-custom-rules/pom.xml

```diff
@@ -14,7 +14,7 @@
 
 	<properties>
 		<sonar.version>6.7</sonar.version>
-		<sonarphp.version>2.12.1.3018</sonarphp.version>
+		<sonarphp.version>2.13.0.3096</sonarphp.version><!-- To be updated when SonarPHP 2.13 is released -->
```

Comment:
@pynicolas but why build fails now?

Reply:
2.13.0.3096 is only a PR build, and it seems that the build of sonar-custom-rules-examples doesn't use repox.sonarsource.com.